config/default.toml

```diff
@@ -35,7 +35,7 @@ DefaultEmailHost = "mozilla.com"
 
     # Length of time for each state leading to the final state
     # to disable a state, make its duration 0 and remove its EventReporter triggers
-    FirstStateDuration = "1m"
+    FirstStateDuration = "5m"
```

Comment:
Okay... this is stupid. So because getting CloudformationResources takes so long by the time they finish the 1m since the Cloudformation was created has elapsed, so it rolls over to SecondState... making the time longer circumvents this